patchwork/step.py

```diff
@@ -90,11 +104,47 @@ def __managed_run(self, *args, **kwargs) -> Any:
     def set_status(self, status: StepStatus, msg: Optional[str] = None):
         if status not in StepStatus.values():
             raise ValueError(f"Invalid status: {status}")
-
+        self.__status = status
+        self.__status_msg = msg
         if status.priority >= self.__status.priority:
             self.__status = status
             self.__status_msg = msg
 
+    def get_key(self):
+        if os.name == 'nt':  # Windows
+            return msvcrt.getch().decode('utf-8')
+        else:  # Linux / macOS
+            fd = sys.stdin.fileno()
+            old_settings = termios.tcgetattr(fd)
+            try:
+                tty.setraw(sys.stdin.fileno())
+                key = sys.stdin.read(1)
+            finally:
+                termios.tcsetattr(fd, termios.TCSADRAIN, old_settings)
+            return key
+        
+    def debug(self, inputs):
+        if inputs.get("debug") is None: 
```

Comment:
This condition only checks for the existence of the `debug` input key. We should ensure that the value of the key `debug` is truthy as well.